### PR TITLE
Wk2assign

### PR DESCRIPTION
**Pull request by rajeshvlega9**
for the second assignment, animation
image
image
image

### CODE REVIEW

File: Week-2/index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Week 2 Assignment 1 - Loren Ipsum</title>
```
Comment (RajeshSivanesan) [inline Week-2/index.html:4]: @rajeshvlega9 Can you create two branches for these two assignments

File: Week-2/index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        </div>
+        <div class="image">
+            <img src="https://cdn.shopify.com/s/files/1/0641/4866/8630/articles/img.png?v=1649204872"/>
+        </div>
+    </div>
+    <div class="contentR">
+        <div>
+            <p>Suspendisse porttitor tincidunt fermentum. Sed rutrum felis nisi, eget semper sapien ornare quis. Duis non quam ac purus tristique commodo. Vivamus auctor scelerisque dui, non ornare purus mattis eget. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin convallis porta nulla at sodales. Donec eget dignissim nisi. Nam fermentum at ligula in convallis.</p>
+        </div>
```
Comment (RajeshSivanesan) [inline Week-2/index.html:26]: @rajeshvlega9 The text and image seems not responsive. Can you look into it ?

Screenshot 2022-07-30 at 11 10 21 AM

File: Week-2/animation.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
+.main {
```
Comment (RajeshSivanesan) [inline Week-2/animation.css:1]: @rajeshvlega9 Animation looks good. Nice work. Can you format the css, it looks not readable

**Review by RajeshSivanesan — COMMENTED** [review]
@rajeshvlega9 Added comments, can you please check ?

File: Week-2/animation.css (hunk 3 above)
Comment (rajeshvlega9) [reply Week-2/animation.css:1]: formatted the animation.css with prettier now

File: Week-2/index.html (hunk 2 above)
Comment (rajeshvlega9) [reply Week-2/index.html:26]: This assignment i have just completed now. i believe now its responsive, please find the screenshot below:
Normal screen image

with smaller screen resolution
image

File: Week-2/index.html (hunk 1 above)
Comment (rajeshvlega9) [reply Week-2/index.html:4]: i will create it from next time. thank you.

**Review by RajeshSivanesan — APPROVED** [review]

**Pull request merged**